File: Resource/51900286-51900712-51900718-51900699/source demo/Window Form/51900286-51900712-51900718-51900699/RestaurentDTO/MonAn.cs
using System.Data;

namespace RestaurentDTO
{
    public class MonAn
    {
        private string _maMonAn;
        private string _tenMonAn;
        private double _donGia;
        private string _donVi;
        private string _moTa;
        private string _moTaNgan;
        private string _hinhAnh;
        private string _maNhom;

        public MonAn(string maMonAn, string tenMonAn, double donGia, string donVi, string moTa, string moTaNgan, string hinhAnh, string maNhom)
        {
            _maMonAn = maMonAn;
            _tenMonAn = tenMonAn;
            _donGia = donGia;
            _donVi = donVi;
            _moTa = moTa;
            _moTaNgan = moTaNgan;
            _hinhAnh = hinhAnh;
            _maNhom = maNhom;
        }

        public MonAn(DataRow dataRow)
        {
            _maMonAn = dataRow["MaMonAn"].ToString();
            _tenMonAn = dataRow["TenMonAn"].ToString();
            _donGia = (double)dataRow["DonGia"];
            _donVi = dataRow["DonVi"].ToString();
            _moTa = dataRow["MoTa"].ToString();
            _moTaNgan = dataRow["MoTaNgan"].ToString();
            _hinhAnh = dataRow["HinhAnh"].ToString();
            _maNhom = dataRow["MaNhom"].ToString();
        }

        public string MaMonAn { get => _maMonAn; set => _maMonAn = value; }
        public string TenMonAn { get => _tenMonAn; set => _tenMonAn = value; }
        public double DonGia { get => _donGia; set => _donGia = value; }
        public string DonVi { get => _donVi; set => _donVi = value; }
        public string MoTa { get => _moTa; set => _moTa = value; }
        public string MoTaNgan { get => _moTaNgan; set => _moTaNgan = value; }
        public string HinhAnh { get => _hinhAnh; set => _hinhAnh = value; }
        public string MaNhom { get => _maNhom; set => _maNhom = value; }
    }
}

File: Dev/Models/Models/MonAn.java
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@Table(name="monan")
public  class MonAn {
    @Id
    @GeneratedValue
    private Long MaMonAn;

    private String TenMonAn;

    private double DonGia;

    private String DonVi;

    private String MoTa;

    private String MoTaNgan;

    private String HinhAnh;

    private String MaNhom;

    @OneToMany(mappedBy = "monan")
    Set<Chitietgoimon> chitietgoimons;
}